Riot/Modules/Analytics/DecryptionFailureTracker.swift

```diff
@@ -103,6 +103,17 @@ class DecryptionFailureTracker: NSObject {
         failure.eventLocalAgeMillis = Int(exactly: eventRelativeAgeMillis)
         failure.trustOwnIdentityAtTimeOfFailure = isSessionVerified
         
+        let myDomain = userId.components(separatedBy: ":")[1]
```

Comment:
Probably safer to do `.last` here instead of `[1]`. Who knows how it can all go wrong and crash.